siliconcompiler/core.py

```diff
@@ -3956,8 +3958,9 @@ def _finalize_run(self, steplist, environment, status={}):
                 if schema.get('flowgraph', flow, step, index, 'status') == NodeStatus.SUCCESS:
                     stat_success = True
             # Merge in manifest if the task was successful.
-            if stat_success:
+            if os.path.isfile(lastcfg):
```

Comment:
Should/does it ever occur that this file doesn't exist? If it shouldn't happen, might be worth an assertion or error check to make sure we don't mask a bug, rather than silently continuing.

Reply:
Yes it can occur and is fine.
The scenario will be that an input node (or the input of the input etc) failed.
In that case we don't write an empty manifest...